### PR TITLE
fix(activity-feed): fix long user names in tasks/comments in Edge

### PR DESCRIPTION
**Pull request by tjgupta**
Follow-up to https://github.com/box/box-ui-elements/pull/1602 that fixes long username wrapping in Edge. Sorry in advance for the browser targeting, but I tried and tried and couldn't find a better way that didn't break other browsers and fixed Edge... :(

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @tjgupta has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/activity-feed/comment/Comment.scss [hunk 1]
```diff
@@ -16,6 +16,14 @@
     }
 }
 
+.bcs-Comment-headline {
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/comment/Comment.scss]: Should we apply this to the UserLink component directly, so it's fixed for all usages?

Comment (tjgupta) [reply src/elements/content-sidebar/activity-feed/comment/Comment.scss]: @jstoffan for some reason that didn't work. It only worked on the `<div>` above it.

Comment (tjgupta) [reply src/elements/content-sidebar/activity-feed/comment/Comment.scss]: Actually it also worked if I did it on the entire `<Media.Body>` too, but that seems worse with potentially more awkward side effects for Edge.

**Pull request merged**